Cargo.toml

```diff
@@ -34,6 +34,8 @@ kakarot-rpc-core = { git = "https://github.com/kkrt-labs/kakarot-rpc.git", rev =
 kakarot-test-utils = { git = "https://github.com/kkrt-labs/kakarot-rpc.git", rev = "ae5e220" }
 
 # Starknet deps
+cairo-vm = "0.8.2"
+blockifier = { package = "blockifier", git = "https://github.com/starkware-libs/blockifier.git", tag = "v0.3.0-rc0" }
```

Comment:
is pinning the tag enough? tag can be overridden

Reply:
I just pinned to the latest tag, I don't think they would override with a breaking change would they? I can pin to a commit if you want